src/helpers/Monitor.cpp

```diff
@@ -25,7 +28,14 @@ void SMonitor::registerListeners() {
     output->setGeometry([this](CCWlOutput* r, int32_t x, int32_t y, int32_t width_mm, int32_t height_mm, int32_t subpixel, const char* make, const char* model,
                                int32_t transform_) { //
         transform = (wl_output_transform)transform_;
-        if (transform == WL_OUTPUT_TRANSFORM_90 || transform == WL_OUTPUT_TRANSFORM_270)
+
+        //if no new mode has been sent before a geometry request, ignore it.
```

Comment:
this is wrong, what if the only thing changing _is_ the transform?

- mode
- geometry 90
- life good
- geometry 0

Reply:
yeah no i just realised that in the shower a minute ago LOL im retarded ill fixy soon